api/inventory/error_response.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package inventory;
+
+option go_package = "inventory";
+
+// ErrorResponse is a message returned on HTTP error.
+message ErrorResponse {
+  string error = 1;
+  string message = 2;
+  int32 code = 3;
+}
```

Comment:
IIUC, we can move that to `header.json` too. We don't need that message in protobuf.